docs/docs/monitoring/analytics.mdx

```diff
@@ -1,19 +1,876 @@
 ---
+id: analytics
 sidebar_label: Analytics
 title: Analytics
-description: Install Rasa Open Source on premises to enable local and customizable Natural Language Understanding and Dialogue Management.
+description:
+abstract: Rasa Pro Analytics is a data pipeline that extracts model, training,
+  and conversation data from the Rasa platform and streams the data to
+  your data warehouse for analysis.
 ---
```

Comment:
I suggest splitting this page in 3 pages:
1. "Getting started with Analytics"
2. "Example queries"
3. "Data structure reference"

@chandrikas @tmbo WDYT?

Reply:
Agree - will be a lot more organised. I get overwhelmed looking at very long pages